go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/openshift/cluster-kube-apiserver-operator
 
-go 1.22.0
```

Comment:
so, it looks like you have opened this pr because [the one](https://github.com/openshift/cluster-kube-apiserver-operator/pull/1773) opened by the art bot [kept](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_cluster-kube-apiserver-operator/1773/pull-ci-openshift-cluster-kube-apiserver-operator-master-verify/1881886989997838336) failing on:
```
Golang version mismatch:
- .ci-operator.yaml: 1.23
- Dockerfile.rhel7: 1.23
- go.mod: 1.22
```


I think that go version indicates the minimum go version that this project supports and the toolchain specifies what binaries/tools must be used, for instance to compile this repo.

Reply:
Yes, I raised the PR to solve that problem. So the go 1.23 is not required for this repo, will recover the orginal 1.22.

Reply:
I misunderstood, will update